packages/market-service/src/foxy/foxy.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  HistoryData,
+  HistoryTimeframe,
+  MarketCapResult,
+  MarketData,
+  MarketDataArgs,
+  PriceHistoryArgs
+} from '@shapeshiftoss/types'
+import dayjs from 'dayjs'
+
+import { MarketService } from '../api'
+import { CoinCapMarketCap } from '../coincap/coincap-types'
+import { RATE_LIMIT_THRESHOLDS_PER_MINUTE } from '../config'
+import { bn } from '../utils/bignumber'
+import { isValidDate } from '../utils/isValidDate'
+import { rateLimitedAxios } from '../utils/rateLimiters'
+
+export const FOXY_CAIP19 = 'eip155:1/erc20:0x61fcabb591d63d00e897a67c64658d376fead816'
```

Comment:
would be nice to get this from a single source of truth, e.g. where we generate assets, rather than risk this contract getting redeployed and missing this

Reply:
use `getFoxyToken`

Reply:
should we have the market-service use asset-service as a peer dependency just for this?